zeebe-benchmark/values.yaml

```diff
@@ -83,6 +83,22 @@ camunda-platform:
         value: "true"
       - name: ZEEBE_BROKER_EXPERIMENTAL_CONSISTENCYCHECKS_ENABLEFOREIGNKEYCHECKS
         value: "true"
+      - name: PYROSCOPE_SERVER_ADDRESS
+        value: "http://pyroscope.pyroscope.svc.cluster.local:4040"
+      - name: PYROSCOPE_APPLICATION_NAME
+        value: "io.camunda.zeebe.broker"
+      - name: PYROSCOPE_LOG_LEVEL
+        value: "debug"
+      - name: PYROSCOPE_FORMAT
+        value: "jfr"
+      - name: PYROSCOPE_PROFILER_EVENT
+        value: "cpu"
+      - name: PYROSCOPE_PROFILER_ALLOC
+        value: "0"
+      - name: PYROSCOPE_PROFILER_LOCK
+        value: "0"
+      - name: PYROSCOPE_LABELS
+        value: "namespace=$(K8S_NAMESPACE), pod=$(K8S_NAME)"
```

Comment:
Shouldn't we move this to the other values file? But you need to copy then other envs as well, but right now we would add them to all our normal benchmarks :thinking:

Reply:
That'd work but increase the risk of env variables drifting apart, right? Personally I'd rather have a couple of useless env variables set than having to remember adjusting _two_ value files. But no strong opinion, your call